src/Controller/HomeController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: varloteaux
+ * Date: 10/10/18
+ * Time: 13:37
+ */
+
+namespace Controller;
+
+use Model\HomeManager;
+
+class HomeController extends Controller
+{
+    public function home()
+    {
+        $itemActive = ["home" => "active","dish" => "","grower" => ""];
```

Comment:
c'est quoi dish et grower ? si ces parties sont "à venir", ces variables ne doivent pas encore etre dans le twig.
Et de toute façon, il faudrait plutot dans ce cas faire un `         return $this->twig->render('Home/home.html.twig', ["home" => "active","dish" => "","grower" => ""]); `  directement...
mais là c'est à retravailler...